src/app/item-page/simple/field-components/preview-section/file-description/file-description.component.spec.ts

```diff
@@ -58,7 +56,7 @@ describe('FileDescriptionComponent', () => {
     fileInput.id = 123;
     fileInput.name = 'testFile';
     fileInput.description = 'test description';
-    fileInput.fileSize = '5MB';
+    fileInput.fileSize = 2048;
```

Comment:
this is not 5MB like it was. Is it ok that this is 2MB?

Reply:
It is OK, I just wanted to put there some value.